src/main/java/com/example/demo/HelloPage.java

```diff
@@ -0,0 +1,13 @@
+package com.example.demo;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+
+public class HelloPage {
+    @GetMapping("/hello/Mark")
```

Comment:
Mark should be a parameter, it is a good practise

Reply:
Ok, I'll do it in next pull request if necessary.